frontend/src/queries/nodes/WebOverview/WebOverview.tsx

```diff
@@ -148,6 +148,19 @@ const formatUnit = (x: number, options?: { precise?: boolean }): string => {
     return humanFriendlyLargeNumber(x)
 }
 
+const getCurrencySymbol = (currency: string): { symbol: string; isPrefix: boolean } => {
+    const formatter = new Intl.NumberFormat('en-US', {
+        style: 'currency',
+        currency: currency,
+    })
+    const parts = formatter.formatToParts(0)
+    const symbol = parts.find((part) => part.type === 'currency')?.value
+
+    const isPrefix = symbol ? parts[0].type === 'currency' : true
+
+    return { symbol: symbol ?? currency, isPrefix }
+}
```

Comment:
I wonder if there's any currency where the currency symbol is in the middle? I believe I've seen some country I visited write their currency like this: `1$500`. Doing some research

Reply:
Lol, forget it, the Brazilian currency in the 1920s did it like that, so that's where I got that from. I'm happy we changed it :)